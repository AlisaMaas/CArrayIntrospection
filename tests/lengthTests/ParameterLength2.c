/**
* This tests traditional what happens if two parameters are the length.
**/
int foo(int* array, int x, int y){
	int sum = 0;
	for(int i = 0;  x > i;  i++){
	    sum += array[i];
	}
	for(int i = 0;  y > i;  i++){
	    sum += array[i];
	}
	return sum;
}
