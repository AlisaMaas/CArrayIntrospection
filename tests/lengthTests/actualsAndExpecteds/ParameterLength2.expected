Analyzing foo
	Argument array has unknown max index.
	Argument x has unknown max index.
	Argument y has unknown max index.
